register.php

```diff
@@ -28,6 +28,24 @@
 			}
 		}
 		
+		if(isset($_POST['g-recaptcha-response'])){
+          $captcha=$_POST['g-recaptcha-response'];
+        }
+		
+		if(!$captcha){
+          echo '<h2>Please check the the captcha form.</h2>';
+          exit;
+        }
+        $secretKey = "Put your secret key here";
```

Comment:
Well, if we can get it to work properly without cURL, then that is the way to go. We want to be as dependency free as possible.